SQL/SQLite/src/Utility.cpp

```diff
@@ -59,15 +59,18 @@ Utility::TypeMap Utility::_types;
 Poco::Mutex Utility::_mutex;
 
 
-Utility::SQLiteMutex::SQLiteMutex(sqlite3* pDB): _pMutex(sqlite3_db_mutex(pDB))
+Utility::SQLiteMutex::SQLiteMutex(sqlite3* pDB)
 {
-	sqlite3_mutex_enter(_pMutex);
+	_pMutex = (pDB) ? sqlite3_db_mutex(pDB) : NULL;
```

Comment:
a couple of things:
1) this is ok, but please leave it in initialization
2) use 0, not NULL

Reply:
strictly speaking `nullptr` would be the right thing to do, but since 1.x releases do not yet unconditionally support c++11, let's have 0

Reply:
Added the changes. Thanks for pointing out!